consumer_pool.go

```diff
@@ -25,14 +26,14 @@ func NewConsumerPool(uris []string, tubes []string, config Config) (*ConsumerPoo
 
 	pool := &ConsumerPool{C: config.jobC, config: config, stop: make(chan struct{})}
 	for _, uri := range multiply(uris, config.Multiply) {
-		consumer, err := NewConsumer(uri, tubes, config)
-		if err != nil {
-			pool.Stop()
-			return nil, err
-		}
+		consumer, _ := NewConsumer(uri, tubes, config)
 
 		pool.consumers = append(pool.consumers, consumer)
```

Comment:
I think this line needs to be conditional of the `NewConsumer()` call not erroring out.